src/configStore.ts

```diff
@@ -6,38 +6,49 @@
  */
 
 import { createStore, applyMiddleware, compose } from 'redux'
-import { rootReducer } from './modules/reducers'
+import createRootReducer from './modules/reducers'
+import { createBrowserHistory } from 'history'
 import thunk from 'redux-thunk'
 import axiosMiddleware from 'redux-axios-middleware'
+import { routerMiddleware } from 'connected-react-router'
 import axios from './api/axios'
 
-const isProduction = process.env.NODE_ENV === 'production'
-
-// Config redux devtool in development
+export const history = createBrowserHistory()
```

Comment:
I would prefer we export all at the end of the file. 
`const history = createBrowserHistory()`